source/content/guides/drupal-8-composer-no-ci.md

```diff
@@ -168,7 +168,13 @@ Normally the next step would go through the standard Drupal installation. But si
    git push --force
    ```
 
-   **Note:** the `vendor` directory is being committed to Pantheon. This is because Pantheon needs the full site artifact. If you prefer to ignore the `vendor` directory then take a look at [our Build Tools guide](/guides/build-tools/) for documentation on the more advanced automated workflow with a build step.
+  <Alert type="info" title="Note">
+  
+  The `vendor` directory is being committed to Pantheon. This is because Pantheon needs the full site artifact. If you prefer to ignore the `vendor` directory then take a look at [our Build Tools guide](/guides/build-tools/) for documentation on the more advanced automated workflow with a build step.
+  
+  With Build Tools, you can also use `terminus build:env:push` for this step.
```

Comment:
We should also include a brief explanation here. 

- Including a Composer dependency from "source" rather than "dist" will create a .git directory inside the dependency's directory. Composer always installs "dev" versions from source instead of dist.
- The presence of a .git directory will prevent the dependency from being committed, which will lead to problems.
- The .git directories must therefore be removed before committing.
- `terminus build:env:push` will remove the .git directories, commit all changed files, and push to origin.

We should give more complete instructions on using build:env:push.

```
terminus build:env:push site.env --message='Commit message'
```

We should include a link to the docs page on installing Terminus and the Terminus Build Tools plugin.